field_value_parser.py

```diff
@@ -14,6 +14,36 @@
 from yaml import FullLoader, load
 
 
+def _dict_key_val_str(key, value):
```

Comment:
Pourquoi ne pas ajouter des annotations de types optionnels?

```suggestion
def _dict_key_val_str(key: Any, value: Any) -> str:
```

où `Any` vient de `from typing import Any`